tests/unit/helpers.js

```diff
@@ -85,6 +85,9 @@ const mountOptionsWithRouter = function (options = {}) {
 const mountOptionsWithApollo = function (options = {}) {
   return Object.assign(options, {
     apolloProvider: createProvider(),
+    stubs: {
```

Comment:
There are other methods for this..
This is nothing to do with apollo

You can chain options things like so: `options = mountOptionsWithApollo(mountOptionsWithRouter({}))`